core/systems/crew/model.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Systems.Crew.Model do
+  @moduledoc """
+  The schema for a group of participants with (assigned) tasks
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "crews" do
+    field(:reference_type, :string)
```

Comment:
Maybe make this an enum? That would allow us to document expected target types.

https://hexdocs.pm/ecto/Ecto.Enum.html